fiddler/assets/dashboards/fiddler_overview.json

```diff
@@ -0,0 +1 @@
1"}],"conditional_formats":[{"comparator":"<","palette":"white_on_green","value":0.2},{"comparator":"<","palette":"white_on_yellow","value":0.5},{"comparator":">","palette":"white_on_red","value":0.5}],"response_format":"scalar","queries":[{"query":"avg:histogram_drift_probability_churn{*}","data_source":"metrics","name":"query1","aggregator":"avg"}]}],"autoscale":true,"custom_links":[{"link":"https://demo.fiddler.ai/projects/bank_churn/models/random_forest_433/monitor?bin_key=ONE_DAY&start_time=1658076858639&end_time=1660668858639&drift_field=probability_churn&drift_metric=jsd&integ_field=__ANY__&tab=Data+Drift&current_decision=decisions&current_acc_metric=accuracy&bin_start=1658102400000","label":"https://demo.fiddler.ai/projects/bank_churn/models/random_forest_431/monitor?bin_key=ONE_DAY&start_time=1646287200000&end_time=1649912399000&drift_field=probability_churn&integ_field=__ANY__&tab=Data+Drift&current_decision=decisions&current_acc_metric=accuracy&bin_start=1649376000000"}],"precision":2}},{"id":1221242763110366,"layout":{"x":71,"y":10,"width":33,"height":15},"definition":{"title":"Traffic","title_size":"16","title_align":"left","type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"conditional_formats":[{"comparator":">","palette":"white_on_green","value":200},{"comparator":"<","palette":"white_on_red","value":105},{"comparator":">","palette":"white_on_yellow","value":105}],"response_format":"scalar","queries":[{"query":"sum:traffic_count{project:bank_churn}.rollup(count, 3600)","data_source":"metrics","name":"query1","aggregator":"avg"}]}],"autoscale":true,"custom_links":[{"link":"https://demo.fiddler.ai/projects/bank_churn/models/random_forest_433/monitor?bin_key=ONE_DAY&start_time=1658076858639&end_time=1660668858639&drift_field=probability_churn&drift_metric=jsd&integ_field=__ANY__&tab=Service+Metrics&current_decision=decisions&current_acc_metric=accuracy&bin_start=1658102400000","label":"https://demo.fiddler.ai/projects/bank_churn/models/random_forest_431/monitor?bin_key=ONE_DAY&start_time=1646287200000&end_time=1649912399000&drift_field=probability_churn&integ_field=__ANY__&tab=Service+Metrics&current_decision=decisions&current_acc_metric=accuracy&bin_start=1649376000000"}],"precision":2}},{"id":3293231302427438,"layout":{"x":3,"y":0,"width":23,"height":9},"definition":{"type":"image","url":"https://global-uploads.webflow.com/5e067beb4c88a64e31622d4b/623e2b19056c2266cff2d854_fiddler-ai-logo.svg","sizing":"contain","margin":"md","has_background":true,"has_border":false,"vertical_align":"center","horizontal_align":"center"}},{"id":7130745675982288,"layout":{"x":27,"y":0,"width":77,"height":9},"definition":{"type":"note","content":"Building trust into AI","background_color":"white","font_size":"56","text_align":"center","vertical_align":"center","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true}}],"template_variables":[],"layout_type":"free","is_read_only":false,"notify_list":[],"id":"kcy-if8-t24"}
```

Comment:
@rajeevthefiddler What are the `https://demo.fiddler.ai/*` custom links linking to? It looks like they're linking to a fiddler account? Can we please remove the links?

Reply:
Looks like the `project:bank_churn` tag is hardcoded. Can we please remove hard-coded tags? You can create template variables for the tags instead: https://docs.datadoghq.com/dashboards/template_variables/#pagetitle

Reply:
Hey @rajeevthefiddler, some of the metrics in the dashboard aren't present in the metadata.csv, like `fiddler.histogram_drift_probability_churn`, `fiddler.recall`, `fiddler.fpr`, and a few others. Can you please add them to the metadata.csv?

Reply:
@fanny-jiang - I have added the possible metrics that can show up in the metadata.csv file. The fiddler.histogram_drift_pred_is_fraud and histogram_drift_probability_churn are going to be model specific. And I have it in the dashboard as an example, just to show the user how to modify the dashboard depending on the model that they bring in. Is there any problem leaving it there? If you want me to remove it, I can. But I will have to delete those charts from the dashboard which I dont want to if possible.